### PR TITLE
Add "vscode" icon to docs sidebar

### PR DESCRIPTION
**Pull request by julieg18**
See https://github.com/iterative/dvc.org/pull/3665#discussion_r897094431.

### CODE REVIEW

File: src/@dvcorg/gatsby-theme-iterative/components/Documentation/Layout/SidebarMenu/icons/index.ts [hunk 1]
```diff
@@ -3,11 +3,13 @@ import defaultIcons, {
 } from '@dvcorg/gatsby-theme-iterative/src/components/Documentation/Layout/SidebarMenu/icons'
 import { ReactComponent as CMLIcon } from './cml-icon.svg'
 import { ReactComponent as StudioIcon } from './studio-icon.svg'
+import { ReactComponent as VsCodeIcon } from '../../../../../../../../static/img/vscode-icon.svg'
```
Comment (julieg18) [inline src/@dvcorg/gatsby-theme-iterative/components/Documentation/Layout/SidebarMenu/icons/index.ts:6]: Would look better in a grayed-out version, but we can only use the officially logo legally :/

**Review by jorgeorpinel — DISMISSED** [review]
I don't think we can use their icon here at all actually.

image

> https://code.visualstudio.com/brand

Can we just get a grayed-out version of the DVC logo in there for now, instead? So the entry stands out.

**Comment by shcheklein** [comment]
@julieg18 @jorgeorpinel let's keep VS Code but grey it out, similar to Studio

**Comment by julieg18** [comment]
> @julieg18 @jorgeorpinel let's keep VS Code but grey it out, similar to Studio

I thought we couldn't modify the logo? https://code.visualstudio.com/brand

**Comment by shcheklein** [comment]
@julieg18 yes, but we can't keep it colorful, I think it's fine if use it as silhouette/shape icon, especially in case like this - we are not branding any product, etc.

At least you can get it from the https://simpleicons.org/?q=vs%20code and use it - they don't restrict it.

Or the way we do GH icons for navigation in different colors a bit for better contrast.

Hard for me to tell exactly, but it feels fine tbh. Clearly, we can't keep it "blue", so even for them it would be better to have it in full grey.

**Review by rogermparent — APPROVED** [review]
Ignoring the question of being allowed to use the logo, the code looks good!

**Review by shcheklein — APPROVED** [review]

**Pull request merged**